final_task/rss_reader/cmd_line_parser.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3.8
+
+import argparse
+import json
+import logging
+
+
+# Make a parser for parsing exact arguments out of sys.argv.
+PARSER = argparse.ArgumentParser(description="Performs a variety of operations on a file.")
+
+PARSER.add_argument('source', help='RSS URL')
+PARSER.add_argument('--version', action="store_true", help="Print version info")
+PARSER.add_argument('--json', action="store_true", help="Print result as JSON in stdout")
+PARSER.add_argument('--verbose', action="store_true", help="Outputs verbose status messages")
+PARSER.add_argument('--limit', type=int, default=None, help="Limit news topics if this parameter provided")
```

Comment:
It is better to make that a function. Also It is better to avoid using global variables whenever you can